src/cryptography/hazmat/backends/openssl/ciphers.py

```diff
@@ -55,6 +55,8 @@ def __init__(self, backend, cipher, mode, operation):
                 _Reasons.UNSUPPORTED_CIPHER
             )
 
+        utils._check_bytes("{0} key".format(cipher.name), cipher.key)
```

Comment:
I think it'd make more sense for this code to live in `algorithms.py`, specifically in `_verify_key_size`, which is already used by all the algorithm classes, and which will provide an earlier error message

Reply:
You're right. I will change it. What about tests?

Reply:
They should be changed to verify the `AES` constructor errors on non-bytes arguments